sidebars.js

```diff
@@ -1024,22 +1024,22 @@ module.exports = {
                                 'secure-connections/sauce-connect-5/installation/kubernetes',
                             ],
                         },
-                        'secure-connections/sauce-connect-5/logging',
-                        'secure-connections/sauce-connect-5/monitoring',
-                        'secure-connections/sauce-connect-5/inspecting',
                         'secure-connections/sauce-connect-5/system-requirements',
                         {
                             type: 'category',
-                            label: 'Setup and Configuration',
+                            label: 'Guides',
                             collapsed: true,
                             items: [
-                                'secure-connections/sauce-connect-5/operation/overview',
-                                'secure-connections/sauce-connect-5/operation/configuration',
-                                'secure-connections/sauce-connect-5/operation/proxies',
-                                'secure-connections/sauce-connect-5/operation/api-server',
-                                'secure-connections/sauce-connect-5/operation/tunnel-pool',
-                                'secure-connections/sauce-connect-5/operation/ci-cd-integration',
-                                'secure-connections/sauce-connect-5/operation/readiness-checks',
+                                'secure-connections/sauce-connect-5/guides/overview',
```

Comment:
We could take down overview, since we decided to move it to quickstart

Reply:
I thought about doing it in a separate MR. Once we have important content from overview moved to quickstart